hw/top_earlgrey/data/chip_conn_testplan.hjson

```diff
@@ -0,0 +1,776 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "chip_conn"
+
+  testpoints: [
+
+    /////////////////////////
+    // AON_TIMER resets    //
+    /////////////////////////
+    {
+      name: AON_TIMER_RST
+      desc: ''Verify rstmgr's resets_o is connected to aon_timer's reset port.''
+      milestone: V1
+      tests: ["aon_timer_rst.csv"]
+    }
+    {
+      name: AON_TIMER_RST_AON
+      desc: ''Verify rstmgr's resets_o is connected to aon_timer's aon-reset port.''
+      milestone: V1
+      tests: ["aon_timer_rst.csv"]
+    }
+
+    /////////////////////////
+    // AST memory          //
+    /////////////////////////
+    {
+      name: AST_DFT_SPI_DEVICE_RAM_2P_CFG
+      desc: '''Verify ast model's configuration bits are connected to spi_device's dual port RAM
```

Comment:
can we merge the 1 to N connection to one testpoint, which may be easier to read and reduce some entries?
```
      desc: '''Verify ast model's dual port configuration bits are connected to the dual port RAMs in the following blocks
                  - spi_device
                  - usbdev
```

Reply:
Thanks weicai. Sounds good, will merge them.